src/Servers/Kestrel/Transport.Quic/src/QuicTransportOptions.cs

```diff
@@ -23,15 +23,10 @@ public class QuicTransportOptions
         /// </summary>
         public ushort MaxUnidirectionalStreamCount { get; set; } = 10;
 
-        /// <summary>
-        /// The Application Layer Protocol Negotiation string.
-        /// </summary>
-        public string? Alpn { get; set; }
-
         /// <summary>
         /// Sets the idle timeout for connections and streams.
         /// </summary>
-        public TimeSpan IdleTimeout { get; set; }
+        public TimeSpan IdleTimeout { get; set; } = TimeSpan.FromMinutes(2);
```

Comment:
What timeout do sockets have? I recall we changed it slightly. Probably should be consistent.

Reply:
Sockets don't have this timeout, the closest match is KestrelServerLimits.KeepAliveTimeout at 130s. I'll use the same value for now and file an issue to sync them.